build/utils/gulp.ts

```diff
@@ -1,8 +1,14 @@
 import { run } from './process'
 import type { TaskFunction } from 'gulp'
 
-export const withTaskName = <T extends TaskFunction>(name: string, fn: T) =>
-  Object.assign(fn, { displayName: name })
+export const withTaskName = <T extends TaskFunction>(
+  name: string,
+  fn: T,
+  shell = false
+) => {
+  if (shell) return Object.assign(fn, { displayName: `shell task ${name}` })
+  return Object.assign(fn, { displayName: name })
+}
 
 export const runTask = (name: string) =>
-  withTaskName(name, () => run(`pnpm run build ${name}`))
+  withTaskName(name, () => run(`pnpm run build ${name}`), true)
```

Comment:
```suggestion
  withTaskName(`shellTask:${name}`, () => run(`pnpm run build ${name}`))
```

Reply:
Thank you for your suggestion. I'll deal with it